### PR TITLE
Some more InSpec coverage: default, web and _services

### PR DESCRIPTION
**Pull request by jvrplmlmn**
Related with https://github.com/StackStorm/chef-stackstorm/issues/29

### CODE REVIEW

File: test/recipes/default/default_test.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# # encoding: utf-8
+
+# Inspec test for recipe stackstorm::_packages
```
Comment (shortdudey123) [inline test/recipes/default/default_test.rb]: `_packages` copy/paste

**Review by shortdudey123 — CHANGES_REQUESTED** [review]

File: test/recipes/default/default_test.rb (hunk 1 above)
Comment (jvrplmlmn) [reply test/recipes/default/default_test.rb]: Fixed in 5c8cc2104e6db12d7c3a87a150e1cc503c75ea4c

**Comment by jvrplmlmn** [comment]
Travis is happy with the changes. @shortdudey123 I've already addressed your comment, thanks for catching it up!

CC @armab

**Review by arm4b — APPROVED** [review]
:+1:

**Review by shortdudey123 — APPROVED** [review]

**Pull request merged**